src/System.Private.CoreLib/Common/System/SR.cs

```diff
@@ -149,7 +150,7 @@ private static string InternalGetResourceString(string key)
             }
         }
 
-        internal static string Format(IFormatProvider provider, string resourceFormat, params object[] args)
+        internal static string Format(IFormatProvider provider, string resourceFormat, params object?[] args)
```

Comment:
I guess there is no need to mark args as nullable since if a caller passes down `null` it will treat it as an array with one element?

Reply:
Ah.  This should be marked as nullable.